ramanujantools/cmf/cmf.py

```diff
@@ -248,10 +248,10 @@ def replace(i):
         return trajectory_matrix({axis: replace(axis) for axis in trajectory.keys()})
 
     @multimethod
-    def walk(
+    def walk(  # noqa: F811
         self,
         trajectory: dict,
-        iterations: Collection[int],
+        iterations: List,
```

Comment:
```suggestion
        iterations: List[int],
```

Reply:
So, the reason for this is because of the change of trajectory matrix. For CMFs, we don't have a nice trajectory matrix to use now, so the code becomes very complicated as is. I figured it's best if we don't have the added complication of iterations as list, and would rather implement it as a single function and then patch together the spaces in the list functionality (which was also my first instinct previously but I was lazy)